core/src/test/java/org/hibernate/ogm/backendtck/jpa/JPAAndJTAViaContainerAPITest.java

```diff
@@ -1,48 +0,0 @@
-/*
- * Hibernate OGM, Domain model persistence for NoSQL datastores
- *
- * License: GNU Lesser General Public License (LGPL), version 2.1 or later
- * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
- */
-package org.hibernate.ogm.backendtck.jpa;
-
-import javax.persistence.EntityManager;
-
-import org.junit.Test;
-import org.hibernate.ogm.utils.jpa.JpaTestCase;
-
-import static org.fest.assertions.Assertions.assertThat;
-
-/**
- * Test that PersistenceProvider#createContainerEntityManagerFactory work properly in a JTA environment
- * @author Emmanuel Bernard &lt;emmanuel@hibernate.org&gt;
- */
-public class JPAAndJTAViaContainerAPITest extends JpaTestCase {
```

Comment:
This one is not needed any more at all? It seems not to do what the description said anyways?